### PR TITLE
Update status, library and location data by copying from discovery and cross checking against NEOS member directory

### LINKED ISSUE #173: [NEOS Discovery] Verify all locations, libraries, and statuses
As with UAL Discovery, we need to verify that all locations, libraries and statues are correctly specified in their YAML files. For example, read on site status is not coming through:

https://catalogue.neoslibraries.ca/catalog/7041306

### PR DESCRIPTION
**Pull request by pgwillia**
I copied [statuses.yml](https://github.com/ualbertalib/discovery/blob/master/config/statuses.yml) directly and copy/pasted from [locations.yml](https://github.com/ualbertalib/discovery/blob/master/config/locations.yml) and added urls back in.

closes #173

### CODE REVIEW

File: config/locations.yml [hunk 1]
```diff
@@ -5,94 +5,95 @@ aginternet:
   name: "Alberta Government Library - Internet"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
 aglcap:  
-  name: "Alberta Government Library"
+  name: "Alberta Government Library - Capital Boulevard"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
```
Comment (pgwillia) [inline config/locations.yml]: This is the location listed in the url.  Not sure if this should be applied to all the other 'Alberta Government Library' locations that weren't previously included.

File: config/locations.yml [hunk 2]
```diff
@@ -103,165 +104,138 @@ covenantgn:
...
-cross:   
-  name: "Alberta Health Services - Cross Cancer Institute"
-  locationurl: 
-cuc:   
-  name: "Burman University"
+cross:
+  name: "Cross Cancer Institute"
+  locationurl:
+cuc:
+  name: "Canadian University College"
```
Comment (pgwillia) [inline config/locations.yml]: Not sure if this is correct.  The url is "Burman University".  Also duplicated in `burman`

Comment (seanluyk) [reply config/locations.yml]: @pgwillia I was mistaken, this code is no longer used - it's not in the Sirsi libraries. The current/correct code is Burman

File: config/locations.yml [hunk 3]
```diff
@@ -103,165 +104,138 @@ covenantgn:
...
-grmacsoc:   
-  name: "MacEwan University - South CampusOn the MoveExpect Delay"
-  locationurl: "https://library.macewan.ca/​"
-grmactsv:   
-  name: "MacEwan University - Technical Services"
+grmactsv:
+  name: "Grant MacEwan University - Technical Services"
   locationurl: "https://library.macewan.ca/"
-grmacewan:   
-  name: "MacEwan University - City Centre Campus"
```
Comment (pgwillia) [inline config/locations.yml]: https://www.neoslibraries.ca/member-libraries/macewan/ and https://library.macewan.ca/ appear to be branded "MacEwan".  Discovery has this as "Grant MacEwan." Which is correct?

Comment (seanluyk) [reply config/locations.yml]: Good catch, @pgwillia, all MacEwan locations should be MacEwan University

File: config/locations.yml [hunk 4]
```diff
@@ -1,148 +1,137 @@
...
+  name: "Alberta Health Services - Alberta Children's Hospital"
+  locationurl: "http://krs.libguides.com/aboutus/ach"
+ahsahe:
   name: "Alberta Health Services - Alberta Hospital Edmonton"
   locationurl: 
 ahscenten:  
   name: "Alberta Health Services - Centennial Centre"
   locationurl: 
-ahserc:  
-  name: "Alberta Health Services - Education Resource Centre"
```
Comment (pgwillia) [inline config/locations.yml]: This was more specific than what was in Discovery [Alberta Health Services].  Which is correct?

File: config/locations.yml [hunk 5]
```diff
@@ -1,148 +1,137 @@
...
-  locationurl: 
-ahsmhrh:  
-  name: "Alberta Health Services - Medicine Hat Regional Hospital"
```
Comment (pgwillia) [inline config/locations.yml]: Not in Discovery.

File: config/locations.yml [hunk 6]
```diff
@@ -1,148 +1,137 @@
...
-  locationurl: 
-ahsqeii:  
-  name: "Alberta Health Services - Queen Elizabeth II Hospital"
```
Comment (pgwillia) [inline config/locations.yml]: Not in Discovery.

File: config/locations.yml [hunk 7]
```diff
@@ -1,148 +1,137 @@
...
+  name: "Alberta Health Services - Kaye Edmonton Clinic"
+  locationurl: "https://krs.libguides.com/aboutus/kec"
+ahsreddr:
   name: "Alberta Health Services - Red Deer Regional Hospital"
   locationurl: "https://krs.libguides.com/aboutus/rdrh"
-ahssturg:  
-  name: "Alberta Health Services - SturgeonIn TransitionExpect Delays"
-  locationurl: 
-ahsplc:  
-  name: "Alberta Health Services - Peter Lougheed Centre"
```
Comment (pgwillia) [inline config/locations.yml]: Not in Discovery.

File: config/locations.yml [hunk 8]
```diff
@@ -1,148 +1,137 @@
...
   locationurl: "https://krs.libguides.com/aboutus/rgh"
-ahstbcc:  
+ahssturg:
+  name: "Alberta Health Services - Sturgeon Community Hospital"
+  locationurl:
+ahstbcc:
   name: "Alberta Health Services - Tom Baker Cancer Centre"
-  locationurl: 
-ahswhc:  
-  name: "Alberta Health Services - Women's Health Centre Alberta Children's Hospital"
```
Comment (pgwillia) [inline config/locations.yml]: Not in Discovery.

File: config/locations.yml [hunk 9]
```diff
@@ -1,148 +1,137 @@
...
-  name: "Alberta Health Services - Centre 15"
-  locationurl: 
-ahsctr:   
-  name: "Alberta Health Services - Centre 15"
-  locationurl: 
-15:   
-  name: "Alberta Health Services - Centre 15"
-  locationurl: 
-ahscross:   
-  name: "Alberta Health Services - Cross Cancer Institute"
```
Comment (pgwillia) [inline config/locations.yml]: not in Discovery.

File: config/locations.yml [hunk 10]
```diff
@@ -1,148 +1,137 @@
...
+ahsint:
   name: "Alberta Health Services/Covenant Health - Internet"
   locationurl: "https://krs.libguides.com/home"
-ahskec:   
-  name: "Alberta Health Services - Kaye Edmonton Clinic"
-  locationurl: "https://krs.libguides.com/aboutus/kec"
 ahsglenrse:   
   name: "Alberta Health Services - Glenrose Rehabilitation Hospital"
   locationurl: "https://krs.libguides.com/aboutus/grh"
-ahsglenrs:
```
Comment (pgwillia) [inline config/locations.yml]: different code in Discovery.

File: config/locations.yml [hunk 11]
```diff
@@ -1,148 +1,137 @@
...
   locationurl: "​http://www.concordiasem.ab.ca/research/library_information.php​"
 covenantgn: 
   name: "Covenant Health - Grey Nuns Community Hospital"
-  locationurl: ​"https://www.covenanthealth.ca/corporate-information/library-services"
+  locationurl: "​https://www.covenanthealth.ca/corporate-information/library-services​"
 covenantmh:   
   name: "Covenant Health - Misericordia Community Hospital"
   locationurl: "​https://www.covenanthealth.ca/corporate-information/library-services​"
-cross:   
-  name: "Alberta Health Services - Cross Cancer Institute"
```
Comment (pgwillia) [inline config/locations.yml]: Just "Cross Cancer Institute" in Discovery.

File: config/locations.yml [hunk 12]
```diff
@@ -151,117 +140,99 @@ justiceca:
 keyano:   
   name: "Keyano College"
   locationurl: "http://www.keyano.ca/Services/Library​"
-king:   
-  name: "The King's University"
+kings:
+  name: "The King's University College"
```
Comment (pgwillia) [inline config/locations.yml]: Looks like Discovery might have this wrong.  "The King's University" is on the webpage.

Comment (nnunn) [reply config/locations.yml]: Yes it's "The King's University"

**Comment by pgwillia** [comment]
Thanks Andrea Dupuis for the information about AHS.

**Comment by seanluyk** [comment]
@pgwillia @nnunn looks like MAP_CABT is map-cabt in statuses.yml.

**Comment by pgwillia** [comment]
Replying to https://github.com/ualbertalib/NEOSDiscovery/pull/208#pullrequestreview-180346467
This is coming from data indexed in Solr.  It's fixed in Discovery https://github.com/ualbertalib/discovery/blob/32bf7e7ac1c438e81bb2a4d6a42f0fd8eb46dc8a/config/SolrMarc/index_scripts/location_facet.bsh#L51 and will be updated with next month's reindex.

**Comment by pgwillia** [comment]
Replying to https://github.com/ualbertalib/NEOSDiscovery/pull/208#pullrequestreview-180349087
I've addressed this in Discovery also.

**Pull request merged**